docs/configure

```diff
@@ -654,7 +657,8 @@ virtual_domains:
   domains = +local_domains
   address_data = ${lookup mysql{\
         select smtp, users.sa_tag*10 AS sa_tag, users.on_spamassassin AS on_spamassassin, \
-        users.uid AS uid, users.gid AS gid, quota from users,domains \
+        users.uid AS uid, users.gid AS gid, quota, domains.spamassassin AS dom_spamassassin \
+        from users,domains \
```

Comment:
I think you could select `users.on_spamassassin OR domains.spamassassin as on_spamassassin`. At least MySQL [seems to support](https://dev.mysql.com/doc/refman/5.5/en/logical-operators.html#operator_or) it, but I'm not 100% sure about [PgSQL](http://www.postgresql.org/docs/current/static/functions-logical.html) (their manual only speaks about boolean variables, although I'm convinced `1` and `0` would qualify as well).